migrations/README.md

```
@@ -0,0 +1,279 @@
+Each BIAN service domain has its own PostgreSQL schemas for business data and audit:
+
+- **`current_account`**: Customer and Account models (BIAN Current Account domain)
+- **`current_account_audit`**: Audit logs for current_account service
+- **`position_keeping`**: Transaction models (BIAN Position Keeping domain)
+- **`position_keeping_audit`**: Audit logs for position_keeping service
+
+Each microservice owns both its business schema and audit schema, ensuring complete service isolation.
+
+### Migration Organization
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)